ansible/templates/topo_t0-isolated.j2

```diff
@@ -0,0 +1,70 @@
+topology:
+  host_interfaces:
+{%- for hostif in hostif_list %}
+    - {{ hostif.port_id }}
+{%- endfor %}
+{%- if vm_list | length == 0 %}
+  VMs: {}
+{%- else %}
+  VMs:
+  {%- for vm in vm_list %}
+    {{ vm.name }}:
+      vlans:
+        - {{ vm.vlans[0] }}
+      vm_offset: {{ vm.vm_offset }}
+  {%- endfor %}
+{%- endif %}
+  DUT:
+    vlan_configs:
+      default_vlan_config: {{ vlan_group_list[0].name }}
+{%- for vlan_group in vlan_group_list %}
+      {{ vlan_group.name }}:
+  {%- for vlan in vlan_group.vlans %}
+        Vlan{{ vlan.id }}:
+          id: {{ vlan.id }}
+          intfs: {{ vlan.port_ids }}
+          prefix: {{ vlan.v4_prefix }}
+          prefix_v6: {{ vlan.v6_prefix }}
+          tag: {{ vlan.id }}
+  {%- endfor %}
+{%- endfor %}
+
+configuration_properties:
+  common:
+    dut_asn: {{ dut.asn }}
+    dut_type: ToRRouter
+    swrole: leaf
+    nhipv4: 10.10.246.254
+    nhipv6: FC0A::FF
+    podset_number: 200
+    tor_number: 16
+    tor_subnet_number: 2
+    max_tor_subnet_number: 16
+    tor_subnet_size: 128
+    spine_asn: 65534
+    leaf_asn_start: 64600
+    tor_asn_start: 65500
+    failure_rate: 0
+
+configuration:
+{%- for vm in vm_list %}
+  {{vm.name}}:
+    properties:
+    - common
+    bgp:
+      asn: {{vm.asn}}
+      peers:
+        {{vm.peer_asn}}:
+          - {{vm.dut_intf_ipv4}}
+          - {{vm.dut_intf_ipv6}}
+    interfaces:
+      Loopback0:
+        ipv4: {{vm.loopback_ipv4}}/32
+        ipv6: {{vm.loopback_ipv6}}/128
+      Ethernet1:
+        ipv4: {{vm.pc_intf_ipv4}}/31
+        ipv6: {{vm.pc_intf_ipv6}}/126
+    bp_interfaces:
```

Comment:
Hi @r12f, I believe this should be `bp_interface`, likewise other references to bp_interfaces would need to be fixed as well (in this PR but also any other similar recent PRs).

Reply:
just to double confirm, this is what I am seeing in existing topologies - t1-64. Does this mean it is also bad?

![image](https://github.com/user-attachments/assets/7fcc7cc5-ad8c-49a2-9232-5353199f91d9)

Reply:
thanks Chris! done!